### ISSUE TITLE
search tags alphabetically when filtering

### ISSUE THREAD

**Issue by klausmcm** [ixti/redmine_tags#217]
I know this plugin is EOL but I'm hoping that this feature could be added to **3.2.2** (I'm on redmine 3.3).

Finding multiple tags when filtering becomes difficult when there are many tags to choose from. Having the tags show in alphabetical order would help significantly I believe. The below shows the only way to filter on multiple tags that I'm aware of.

![image](https://user-images.githubusercontent.com/14914359/81435951-79b1dc80-911d-11ea-9f3d-2f6d2ee0d62b.png)

I'd be very grateful for this feature as I'm not too familiar with ruby. I'm also open to suggestions and workarounds.

Thanks,

**Comment by tainewoo** [comment]
@klausmcm ，
Hi, i think this could be done by replace the select by select2.
i have a plug-in for that.

**Comment by klausmcm** [comment]
Hey, thank you for the suggestion. We are on redmine 3 so I'm hesitant to use a plugin tested for 4. Would what I'm looking for be easy to implement in this plugin? Maybe a `.sort` call somewhere??

**Comment by PowerKiKi** [comment]
ixti/redmine_tags is also only for Redmine 4. Even if this feature gets implemented, you'll have to consider upgrading your Redmine.

**Comment by klausmcm** [comment]
> ixti/redmine_tags is also only for Redmine 4. Even if this feature gets implemented, you'll have to consider upgrading your Redmine.

No, not only, that is why I requested the feature for **v3.2.2**.

**Comment by klausmcm** [comment]
@marius-balteanu Any chance that this request will receive attention? Thanks,

**Comment by sdwolfz** [comment]
@klausmcm your best bet is to fork this repository and implement it yourself, or commision somebody else with ruby experience to do it for you. 

Our focus is to have it integrated into core redmine: https://www.redmine.org/issues/1448 hopefully for version 5. I know upgrading redmine is a difficult task especially if using many plugins with compatibility issues, but still, you'll have to do it at some point... it's only going to get more difficult as time goes on.
